package.json

```diff
@@ -1,152 +1,167 @@
-				"AlphaBETA.phase.showInternal": {
-					"type": "boolean",
-					"scope": "window",
-					"default": true,
-					"markdownDescription": "If enabled, symbols annotated with the `@internal` flag will be highlighted.  Depends on `#AlphaBETA.control.showAnnotations#`"
-				},
-				"AlphaBETA.phase.showAlpha": {
-					"type": "boolean",
-					"scope": "window",
-					"default": true,
-					"markdownDescription": "If enabled, symbols annotated with the `@alpha` flag will be highlighted.  Depends on `#AlphaBETA.control.showAnnotations#`"
-				},
-				"AlphaBETA.phase.showBeta": {
-					"type": "boolean",
-					"scope": "window",
-					"default": true,
-					"markdownDescription": "If enabled, symbols annotated with the `@beta` flag will be highlighted.  Depends on `#AlphaBETA.control.showAnnotations#`"
-				},
-				"AlphaBETA.control.refreshInterval": {
-					"type": "integer",
-					"scope": "window",
-					"default": 10,
-					"minimum": 2,
-					"maximum": 60,
-					"description": "Specifies the number of seconds that elapse between updates.  If you experience slow performance you can increase this number."
-				},
-				"AlphaBETA.reporting.showInStatusBar": {
-					"type": "boolean",
-					"scope": "window",
-					"default": true,
-					"markdownDescription": "If enabled and the active file is using any `prerelease` symbols, a message will be shown in the status bar.  Clicking the message will open a report listing each usage and its line number."
-				}
-			}
-		},
-		"colors": [
-			{
-				"id": "AlphaBETA.internal",
-				"description": "Color for symbols flagged as @internal",
-				"defaults": {
-					"dark": "#91000D88",
-					"light": "#91000D88",
-					"highContrast": "#91000D88"
-				}
-			},
-			{
-				"id": "AlphaBETA.alpha",
-				"description": "Color for symbols flagged as @alpha",
-				"defaults": {
-					"dark": "#C0BC1788",
-					"light": "#C0BC1788",
-					"highContrast": "#C0BC1788"
-				}
-			},
-			{
-				"id": "AlphaBETA.beta",
-				"description": "Color for symbols flagged as @beta",
-				"defaults": {
-					"dark": "#12A64B88",
-					"light": "#12A64B88",
-					"highContrast": "#12A64B88"
-				}
-			}
-		],
-		"views": {
-			"explorer": [
-				{
-					"id": "AlphaBETA.Prerelease",
-					"type": "tree",
-					"name": "Prelease",
-					"visibility": "collapsed"
-				}
-			]
-		}
-	},
-	"scripts": {
-		"vscode:prepublish": "npm run compile",
-		"compile": "tsc -p ./",
-		"watch": "tsc -watch -p ./",
-		"pretest": "npm run compile && npm run lint",
-		"lint": "eslint src --ext ts",
-		"test": "node ./out/test/runTest.js"
-	},
-	"devDependencies": {
-		"@types/vscode": "^1.55.0",
-		"@types/glob": "^7.1.3",
-		"@types/mocha": "^8.0.4",
-		"@types/node": "^12.11.7",
-		"eslint": "^7.19.0",
-		"@typescript-eslint/eslint-plugin": "^4.14.1",
-		"@typescript-eslint/parser": "^4.14.1",
-		"glob": "^7.1.6",
-		"mocha": "^8.2.1",
-		"vscode-test": "^1.5.0"
-	},
-	"dependencies": {
-		"typescript": "^4.1.3"
-	}
+  "name": "alphabeta",
```

Comment:
What, you didn't like my tabWidth: 24 setting ;)

Reply:
lol sorry i didnt even notice, damn auto-format on save